src/components/PageLayout/PageLayout.tsx

```diff
@@ -39,24 +38,18 @@ export const PageLayout: FC<PageLayoutProps> = ({contentWidth = 2, title = '', c
       </Head>
       <PageTitleContainer.Provider initialState={title}>
         <BannerContainer.Provider>
-          <div className={styles.pageContainer}>
+          <Stack sx={{minHeight: '100vh', position: 'relative'}}>
```

Comment:
len note, kedze si len presunul styly, tak to nie je tvoj issue - `100vh` vie sposobit problemy na mobile, kedze sa nerataju nejake app bary v prehliadaci, takze aj kratsia stranka by sa tvarila chvilu scrollovatelne, chvilu nie, podla toho, ci bar je viditelny... su uz nove veci ako 100dvh a 100svh (https://web.dev/blog/viewport-units) ale najlepsie je aj tak byt schopny pouzit 100% abo daco take 😄 len to musi podporovat parent atd.. neries teraz, just sharing 😄

Reply:
`position: relative` je zbytocny, ak ziadny ani hlbsi child nepouziva `position: absolute`, co neviem. ak by to tu nebolo, bude sa to viazat zase na najblizsieho parenta s `position: relative`, co je mozno ok

Reply:
myslim ze pod tym nieco pred tym bolo, snazil som sa to premazat,ale nevydalo zjavne.

ohadom toho 100%, tak toto je v podstate root celeho layoutu